### DISCUSSION TITLE
'Latest' softlink for ghidra-sre.org downloads?

### DISCUSSION THREAD

**Discussion by kilbooky** [NationalSecurityAgency/ghidra#2669] [Ideas]
Hey folks, 

How hard of an ask is it to implement a "Latest" download link when you go to ghidra-sre.org? I ask because I've been setting up a lot of environments lately and `curl` a specific version of ghidra from the website.

This would help me make sure my scripts are always getting the most recent release. 

Thanks!

**Comment by ryanmkurtz** [comment]
I recently thought this would be very useful too.

**Comment by emteere** [comment]
I've made the changes and submitted them for update.
Hopefully it will be what you need.  If not, just ask.

**Comment by emteere** [comment]
The ghidra-sre.org website has been updated with the Latest link.

**Reply by ryanmkurtz** [reply]
Rather than a button to download the latest, I think what @kilbooky wants is a download URL like `https://ghidra-sre.org/latest` that will resolve to the latest version of Ghidra.  This URL could be put in to scripts that call `wget` or `curl`, and would not have to ever be updated.  This type of URL is what I would have found very useful.

**Reply by ryanmkurtz** [reply]
I think the button is useful too though.  If the button would use the "latest" URL, it too would never have to be updated.

**Reply by xiaoyinl** [reply]
@emteere and @ryanmkurtz I suggest that the Latest link looks like `https://ghidra-sre.org/ghidra_latest_PUBLIC.zip`. If you will release beta or nightly versions, they can be similarly linked as `https://ghidra-sre.org/ghidra_latest_BETA.zip` and `https://ghidra-sre.org/ghidra_latest_DEV.zip`.

**Comment by ryanmkurtz** [comment]
Now that the releases are being distributed via GitHub, you should be able to use the GitHub REST API to script the download of the latest release.  Here is an example:

https://gist.github.com/maxim/6e15aa45ba010ab030c4
